charts/spire/charts/tornjak-ui/templates/tests/test-tornjak-connection.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if eq (.Values.tornjakFrontend.enabled | toString) "true" }}
```

Comment:
This if statement can be removed. As we will disable the entire chart at root chart level.